src/components/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../styles/navbar.css";
+import logo from "../assets/logosound.png";
+
+const Navbar = () => {
+  return (
+    <div className="navbar">
+      <img className="logo" src={logo} alt="logo soundcloud" />
```

Comment:
wouldn't add the actual soundcloud logo, probably easier just to have this be a p tag with the text 'SOUNDCLONE'

Reply:
ok no problem